py4DSTEM/visualize/visualize.py

```diff
@@ -413,5 +413,33 @@ def show_points(ar,x,y,s=1,scale=500,alpha=1,point_color='r',
     else:
         return fig,ax
 
-
+    
+def show_hist(arr, bins = 200, return_hist = False):
```

Comment:
suggest we change the last kwarg to `returnfig` for consistency with the other vis functions, where if this flag is True the figure is returned, and if False it is plotted and not returned